Automattic/jetpack
# My Jetpack: Use Product Detail Table for Jetpack Social

Fixes #29534

## Proposed changes:

* Add tier data for Social in My Jetpack, so that it will render the pricing table instead of the current plugin installation prompt.
* Update mock data for ProductDetailTable stories, to match the Social product data.

### Other information:


## Jetpack product discussion



https://github.com/Automattic/jetpack/issues/29534

## Does this pull request change what data or activity we track or use?



No

## Testing instructions:






* Get a site running this branch with the Jetpack plugin.
* Go to /wp-admin/admin.php?page=my-jetpack#/add-social
* Test both the free and upgrade paths
* Confirm the pricing table content matches the existing table in the Social plugin.

## Screenshots

<img width="1159" alt="Screenshot 2023-05-15 at 10 37 03 AM" src="https://github.com/Automattic/jetpack/assets/10933065/1ae17831-4e82-4e4f-92a7-9d52b5677149">
